core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -170,6 +175,17 @@ public CallOptions withAffinity(Attributes affinity) {
   }
 
   /**
+   * Sets whether the call is not
+   * <a href="https://github.com/grpc/grpc/blob/master/doc/fail_fast.md">fail fast</a>.
+   * Fail fast is the default option for gRPC calls.
+   */
+  public CallOptions withNonFailFast(boolean nonFailFast) {
```

Comment:
Let's mark it with `@ExperimentalApi` since the name of "non fail fast" is still a bit up for debate.

Do we want `withFailFast(void)` and `withWaitForReady(void)`, or do we think the boolean `withNonFailFast(true)` is nicer?